trade-finance-manager-ui/server/controllers/team-checking/index.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response } from 'express';
+import { TeamToCheck } from '../../helpers/team-checking.helper';
+import { asUserSession } from '../../helpers/express-session';
+import { userIsInTeam } from '../../helpers/user';
+
+type RenderTeamsCheckerProps = {
+  teamCombinations: TeamToCheck[];
+  currentPageTeamRestrictions?: TeamToCheck;
+};
+
+export const renderTeamsChecker = (req: Request, res: Response, { teamCombinations, currentPageTeamRestrictions }: RenderTeamsCheckerProps) => {
+  const { user } = asUserSession(req.session);
+
+  const isUserInTeam = currentPageTeamRestrictions && userIsInTeam(user, currentPageTeamRestrictions.teams);
```

Comment:
Might be prudent to get the result from function in a constant so all the variables in compound condition are constants.

```ts
const userIsInTeam = userIsInTeam(user, currentPageTeamRestrictions.teams);
const isUserInTeam = currentPageTeamRestrictions && userIsInTeam;
```